www/src/components/layout/Subheader.tsx

```diff
@@ -82,6 +83,7 @@ export default function Subheader() {
           type="floating"
         />
         <IconFrame
+          data-phid="nav-arrow-fwd"
```

Comment:
I don’t think we’re using abbreviations anywhere else so to keep it consistent I’d use `nav-arrow-forward`.